ddprof-lib/src/main/cpp/ctimer_linux.cpp

```diff
@@ -62,21 +136,52 @@ static int pthread_setspecific_hook(pthread_key_t key, const void *value) {
   }
 }
 
-static void **lookupThreadEntry() {
-  // Depending on Zing version, pthread_setspecific is called either from
-  // libazsys.so or from libjvm.so
-  if (VM::isZing()) {
-    CodeCache *libazsys = Libraries::instance()->findLibraryByName("libazsys");
-    if (libazsys != NULL) {
-      void **entry = libazsys->findImport(im_pthread_setspecific);
-      if (entry != NULL) {
-        return entry;
-      }
-    }
+static Error patch_libraries_for_J9_or_musl() {
+   CodeCache *lib = Libraries::instance()->findJvmLibrary("libj9thr");
```

Comment:
Is this right? 
The `libj9thr` will be available only on J9, regardless of whether we are running on musl or not.